tests/cephfs/cephfs_utilsV1.py

```diff
@@ -235,8 +235,10 @@ def wait_for_mds_process(
                 sudo=True, cmd=f"ceph orch ps | grep {process_name}", check_ec=False
             )
             if client.node.exit_status == 0 and ispresent:
+                sleep(100)
```

Comment:
The method is to wait for for mds service... can we do `ceph orch ps --daemon_type mds --format json`... then iterate through the response to check if the process is running or not.

This way, I believe we can avoid multiple sleeps and ensure it is correctly running.